tests/dragonfly/cluster_test.py

```diff
@@ -2499,3 +2499,56 @@ async def test_migration_timeout_on_sync(df_factory: DflyInstanceFactory, df_see
     await push_config(json.dumps(generate_config(nodes)), [node.admin_client for node in nodes])
 
     assert (await StaticSeeder.capture(nodes[1].client)) == start_capture
+
+
+@pytest.mark.slow
+@dfly_args({"proactor_threads": 4, "cluster_mode": "yes", "dbfilename": "big_snap"})
```

Comment:
lets use tmp_file_name() for dbfilename as we do in other pytests that do snapshot